Set the encoding of virtual consoles to UTF-8.

diff --git a/sutils/MAKEDEV.sh b/sutils/MAKEDEV.sh
index 4277b05..726d3e8 100644
--- a/sutils/MAKEDEV.sh
+++ b/sutils/MAKEDEV.sh
@@ -91,7 +91,7 @@ mkdev() {
       console|com[0-9])
 	st $I root 600 /hurd/term ${DEVDIR}/$I device $I;;
       vcs)
-        st $I root 600 /hurd/console;;
+        st $I root 600 /hurd/console --encoding=UTF-8;;
       tty[1-9][0-9]|tty[1-9])
         st $I root 600 /hurd/term ${DEVDIR}/$I hurdio \
 	   ${DEVDIR}/vcs/`echo $I | sed -e s/tty//`/console;;
