Fix "make dist" in `libthreads'.

diff --git a/libthreads/Makefile b/libthreads/Makefile
index ee86236..1ad688b 100644
--- a/libthreads/Makefile
+++ b/libthreads/Makefile
@@ -57,4 +57,9 @@ ifeq ($(VERSIONING),yes)
 # where ld will read it as a linker script.
 $(libname).so.$(hurd-version): $(srcdir)/$(libname).map
 
+lndist: lndist-map-file
+
+lndist-map-file: $(top_srcdir)/hurd-snap/$(dir)
+	ln $(addprefix $(srcdir)/,$(libname).map) $<
+
 endif
