Fix "make dist" in `exec'.

diff --git a/exec/Makefile b/exec/Makefile
index 9b318e8..a77173a 100644
--- a/exec/Makefile
+++ b/exec/Makefile
@@ -18,12 +18,16 @@
 dir := exec
 makemode := server
 
-SRCS = exec.c main.c hashexec.c hostarch.c unzip.c util.c inflate.c
+SRCS = exec.c main.c hashexec.c hostarch.c \
+       $(gzip-sources) $(bzip2-sources)
 OBJS = main.o hostarch.o exec.o hashexec.o \
        execServer.o exec_startupServer.o \
        $(gzip-objects) $(bzip2-objects)
-gzip-objects = unzip.o util.o inflate.o
-bzip2-objects = do-bunzip2.o
+gzip-sources = unzip.c util.c inflate.c
+gzip-objects = $(gzip-sources:%.c=%.o)
+bzip2-sources = do-bunzip2.c
+bzip2-objects = $(bzip2-sources:%.c=%.o)
+
 LCLHDRS = gzip.h crypt.h tailor.h priv.h execmutations.h
 target = exec
 #targets = exec exec.static
