Fix "make dist" in `pfinet'.

diff --git a/pfinet/Makefile b/pfinet/Makefile
index 9bc97e8..945db21 100644
--- a/pfinet/Makefile
+++ b/pfinet/Makefile
@@ -130,20 +130,6 @@ socket-MIGSFLAGS = -imacros $(srcdir)/mutations.h
 io_S.h ioServer.c socket_S.h socketServer.c: mutations.h
 $(OBJS): config.h
 
-lndist: lndist-linux-inet-files lndist-linux-files lndist-asm-files
-
-lndist-linux-inet-files: $(top_srcdir)/hurd-snap/$(dir)/linux-inet
-	ln $(addprefix $(srcdir)/linux-inet/,$(LINUXSRCS) $(UNUSEDSRC) $(LINUXHDRS)) $<
-
-lndist-linux-files: $(top_srcdir)/hurd-snap/$(dir)/linux
-	ln $(addprefix $(srcdir)/linux/,$(FROBBEDLINUXHEADERS)) $<
-
-lndist-asm-files: $(top_srcdir)/hurd-snap/$(dir)/asm
-	ln $(addprefix $(srcdir)/asm/,$(ASMHEADERS)) $<
-
-$(top_srcdir)/hurd-snap/$(dir)/linux-inet:
-	mkdir $@
-$(top_srcdir)/hurd-snap/$(dir)/linux:
-	mkdir $@
-$(top_srcdir)/hurd-snap/$(dir)/asm:
-	mkdir $@
+lndist: $(top_srcdir)/hurd-snap/$(dir)
+	ln Makefile $(SRCS) $(LCLHDRS) "$<"
+	cp -rv linux-src glue-include "$<"
